embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/spans/SpanRepository.kt

```diff
@@ -101,4 +102,50 @@ class SpanRepository {
     }
 
     private fun notTracked(spanId: String): Boolean = activeSpans[spanId] == null && completedSpans[spanId] == null
```

Comment:
Will you be adding unit tests that verify this works with different edge-case combinations of descendants which might change the traversal logic? Like if a child is no longer active, but the grand child is active?